crates/astria-sequencer/src/transaction/checks.rs

```diff
@@ -197,16 +232,10 @@ async fn sequence_update_fees<S: StateReadExt>(
 }
 
 fn ics20_withdrawal_updates_fees(
-    asset: &asset::Denom,
```

Comment:
Same here - is this functionality no longer necessary or wrong?

Reply:
same as above